docs/products/kafka/kafka-connect/howto/kafka-connect-debezium-tls-pg.md

```diff
@@ -0,0 +1,209 @@
+  [Apache Kafka Connect](/docs/products/kafka/kafka-connect/howto/enable-connect)
+  enabled.
+- Administrative access to a PostgreSQL database with SSL enabled.
+- The following SSL certificates and keys obtained from your PostgreSQL database:
+  - SSL client certificate: The public certificate for client authentication.
+  - SSL root certificate: The certificate of the Certificate Authority (CA) that signed
+    the servers' and clients' certificates.
+  - SSL client key: The client's private key is used to encrypt the data sent to the
+    server.
+  For additional details, see
+  [Certificate requirements](/docs/platform/concepts/tls-ssl-certificates#certificate-requirements).
+
+### For CloudSQL PostgreSQL databases
+
+If you're integrating with a CloudSQL database, perform these additional steps:
+
+- **IP whitelisting**: Whitelist Aiven's IP addresses to allow connections
+  from Apache Kafka Connect to reach your CloudSQL database.
+- **Configure WAL and logical decoding**:
+  - To capture database change events, `set cloudsql.logical_decoding` to `on`.
+  - To configure the Write-Ahead Log (WAL) for logical replication, set
+    `cloudsql.enable_pglogical` to` on`.
+  - Restart the CloudSQL instance to apply changes.
+- **Activate the `pgoutput` extension**: Execute this command in your
+  CloudSQL instance to enable the extension Debezium uses for logical decoding.
+  ```SQL
+  CREATE EXTENSION pgoutput;
+  ```
+- **Verify SSL certificates**: After whitelisting IPs, ensure your SSL configuration is
+  correct by testing the database connection:
+
+  ```bash
+  psql "sslmode=verify-ca \
+    sslrootcert=server-ca.pem \
+    sslcert=client-cert.pem \
+    sslkey=client-key.pem \
+    hostaddr=<YOUR_IP_ADDRESS> \
+    port=5432 \
+    user=<YOUR_USER> \
+    dbname=<YOUR_DB_NAME>"
+  ```
+
+- **Set Debezium output plugin**: Use `pgoutput` or `decoderbufs` as the `plugin.name` in
+  your Debezium connector for logical decoding.
+
+    :::note
+      Starting with Debezium 2.5, the `wal2json` plugin is deprecated. You can use
+      either `pgoutput` or `decoderbufs` as the recommended replacement plugin.
+    :::
+
+## Variables
+
+The following table lists the variables for the configuration steps. Replace them with
+your actual environment values in the provided code snippets:
+
+| Variable | Description |
+|---|---|
+| `<kafka_cluster_name>` | Name of your Aiven for Apache Kafka service |
+| `<kafka_connect_name>` | Name of your Apache Kafka Connect service |
+| `<cloud_provider_and_region>` | Your cloud provider and region identifier |
+| `<plan>` | Service plan for Apache Kafka/Apache Kafka Connect |
+| `<project_name>` | Name of your project in the Aiven platform |
+| `<integration_config>` | JSON configuration for PostgreSQL endpoint |
+| `<integration_endpoint_id>` | ID of the created PostgreSQL endpoint |
+| `<connector_config>` | JSON configuration for Debezium connector |
+
+## Configuration via Aiven CLI
```

Comment:
I'd argue we don't need to say the means to perform a task since we can now use tabs. 
Right now it's just cli, which doesn't bring SEO value here since it's the only choice, and when we get another possibility, it'll be in a tab, not in a new section.

```suggestion
## Configure the integration
```

Reply:
Sure. Updated the heading.